### DISCUSSION TITLE
Is there a way to build without bundle?

### DISCUSSION THREAD

**Discussion by samuelint** [tauri-apps/tauri#10376] [Q&A] [answered]
Hi,

is there a way to build the app, without bundling. Then bundle in a separate step? The CLI arguments does not seem to allow it. How can I achieve that? Is it possible to only execute the bundle step with the cli?

---
My use case is:
1. Create a pre-baked tauri app (without externalBin)
2. Distribute internally the pre-baked app
3. Build the pre-baked app with the externalBin, have some `tauri.conf.json` value changes for this app
4. Build the app -> distribute

**Comment by FabianLars** [comment, marked as answer]
> Is it possible to only execute the bundle step with the cli?

In v2 there's a `tauri bundle` cli command, yes.

That said, your use case sounds like you'll need to rebuild the app itself (due to tauri.conf.json changes) anyway so sounds like you'll need this instead:
1) `tauri build --bundles none` (v1) or `tauri build --no-bundle` (v2)
3) `tauri build`

**Comment by tunaayberk** [comment]
Is there a way also just bundle command? Bundle the build?

**Reply by FabianLars** [reply]
I answered that in my other comment above already, it's `tauri bundle`
